From d3e73c6ce652168963cb10c8284c89f2cf4df16e Mon Sep 17 00:00:00 2001
From: nishans08 <71048600+nishans08@users.noreply.github.com>
Date: Wed, 27 Jul 2022 16:42:04 -0700
Subject: [PATCH 44/85] fixes grammatical errors in README.md

221: You should jump this section and install them from distribution repository instead. -> You should jump to this section and install them from the distribution repository instead.

FIX #7
---
 README.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/README.md b/README.md
index c6e5026..b394e1e 100644
--- a/README.md
+++ b/README.md
@@ -218,7 +218,7 @@ Notes: Fedora 26  libsodium version >= 1.0.12, so you can install via dnf instal
 
 If your system is too old to provide libmbedtls and libsodium (later than **v1.0.8**), you will need to either install those libraries manually or upgrade your system.
 
-If your system provides with those libraries, you **should not** install them from source. You should jump this section and install them from distribution repository instead.
+If your system provides with those libraries, you **should not** install them from source.You should jump to this section and install them from the distribution repository instead.
 
 For some of the distributions, you might install build dependencies like this:
 
-- 
2.25.1

